generated_stamp/Despite_Guaranteeing_AAP's_Landslide_Victory,_There_Are_Sadly_No_Women_On_Its_Cabinet._Why?_stamp.html

```diff
@@ -10,12 +10,12 @@
     <script async custom-element="amp-youtube" src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js"></script>
     <script async custom-element="amp-instagram" src="https://cdn.ampproject.org/v0/amp-instagram-0.1.js"></script>
     <script async custom-element="amp-pinterest" src="https://cdn.ampproject.org/v0/amp-pinterest-0.1.js"></script>
-    <title>This is title of website</title>
```

Comment:
This file can be ignored while review. The name of the file follows the format **title_stamp.html** where if title had spaces, it was replaced with _.